tests/conformance_suite.postman_collection.json

```diff
@@ -4229,6 +4228,54 @@
 					"script": {
 						"type": "text/javascript",
 						"exec": [
+							"// Raw body should exclude all optional items and should contain",
+							"// first alternate version of any 'oneOf' elements defined in the",
```

Comment:
Is there a way to cycle through testing *all* alternate versions of the `oneOf` elements?

Also, is there any intent (assuming possibility) to cycle through each of the optional items? Combinations of optionals would also be desirable, though substantially more complicated, if possible.

Reply:
Yes, and we do that when these tests are complete (see testing for `credentials/issue`